src/Core/Entity/WebhookQueue.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * Ushahidi Webhook Queue
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Platform
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Entity;
+
+use Ushahidi\Core\StaticEntity;
+
+class WebhookQueue extends StaticEntity
```

Comment:
Could we call the entity a 'job' rather than queue? That seems more consistent with usual terminology